config.py

```diff
@@ -13,116 +15,125 @@
 from calibre.utils.config import JSONConfig
 from calibre.gui2 import error_dialog
 
-prefs = JSONConfig('plugins/xray_creator')
+__prefs__ = JSONConfig('plugins/xray_creator')
```

Comment:
You're right. It doesn't like global things being named that way. Global variables need to be capitalized or have the format __*__.